bin/go-rpm-integration

```diff
@@ -208,7 +208,7 @@ for dir in $(\
          ${golistflags[$goipath]}          \
   ) ; do
   pushd "${workroot}/src/${dir}" >/dev/null
-    echo "Testing \"${workroot}/src/${dir}\""
+    echo "## Testing \"${workroot}/src/${dir}\""
```

Comment:
Can you squash this commit with the `Simpler messages` one? Both are semantically equivalent.